lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
-	var visible = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
-	return visible;
+	var outside = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
+	if (outside) {
+		axe.commons.color.incompleteData.set('bgColor', {
+			node: elm,
+			reason: 'elmOutsideParent'
+		});
+	}
+	return outside;
 }
+
+// document.elementsFromPoint misses some elements we need
+// i.e. TR is missing from table elementStack and leaves out bgColor
+// https://github.com/dequelabs/axe-core/issues/273
+function includeMissingElements(elmStack, elm) {
```

Comment:
In theory, yes. But in practice, it only occurs with `TR` elements, which can't be positioned, and inline `LABEL` elements. (It also occurs with `pointer-events: none;`, but I'm not addressing that here)

I could add a visuallyContains check to this but there's a wrinkle, in that `LABEL` ends up missing from the list when itself is being evaluated, so there's nothing to compare against. I think `elmOutsideParent` actually does have a use with this code in place.